kpack-image-builder/controllers/buildreconcilerinfo_controller_test.go

```diff
@@ -360,10 +352,7 @@ var _ = Describe("BuildReconcilerInfoReconciler", Serial, func() {
 			})
 
 			It("sets the buildpacks on the BuildReconcilerInfo", func() {
-				lookupKey := types.NamespacedName{
-					Name:      controllers.BuildReconcilerInfoName,
-					Namespace: rootNamespace.Name,
-				}
+				lookupKey := client.ObjectKeyFromObject(info)
```

Comment:
is info.name the equivalent of controllers.BuildReconcilerInfoName and info.Namespace the equivalent of rootNamespace.Name?

Reply:
Oops, this looks like a mistake. I'll push a fix.